docs/src/start/connect-c8y.md

```diff
@@ -50,6 +50,25 @@ This will set the root certificate path of the Cumulocity IoT.
 In most of the Linux flavors, the certificate will be present in `/etc/ssl/certs`.
 If not found download it from [here](https://www.identrust.com/dst-root-ca-x3).
 
+### Custom domain
+
+If you are using the Cumulocity Iot [custom domain feature](https://cumulocity.com/docs/enterprise-tenant/customization/#domain-name), then you need to set two urls instead of one, as the HTTP and MQTT endpoints on custom domains are different because the custom domain only applies to the HTTP endpoint. The MQTT endpoint must point to the underlying Cumulocity IoT instance.
+
+For example, below shows setting the HTTP and MQTT endpoints:
+
+```sh
+tedge config set c8y.http "custom.example.com"
+tedge config set c8y.mqtt "t12345.cumulocity.com:8883"
+```
+
+:::tip
+If you have `curl` and `jq` installed, then you can check if you are using a custom domain by executing the following command. If the response includes a different domain than the url used in the curl command, then you are using a custom domain, and you should use the output to set the `c8y.mqtt` thin-edge.io setting instead.
```

Comment:
This tip suggests that thin-edge could do the same to infer `c8y.mqtt` from `c8y.http`.

Reply:
Potentially, though we'd have to do some more testing to make sure it works in all cases (where as a doc statement is a lot more forgiving).